bond/bond.py

```diff
@@ -1,119 +1,105 @@
 import requests
-
-BOND_DEVICE_TYPE_CEILING_FAN = "CF"
-BOND_DEVICE_TYPE_FIREPLACE = "FP"
-BOND_DEVICE_TYPE_MOTORIZED_SHADES = "MS"
-BOND_DEVICE_TYPE_GENERIC_DEVICE = "GX"
-
-# Note that Bond Action "Stop" instructs the Bond Bridge to stop sending.
-# There are other actions such as "Hold" that send a message to stop the
-# controlled-device's current action
-BOND_DEVICE_ACTION_STOP = "Stop"
-
-# Relating to Generic Device (GX)
-BOND_DEVICE_ACTION_TURN_ON = "TurnOn"
-BOND_DEVICE_ACTION_TURN_OFF = "TurnOff"
-BOND_DEVICE_ACTION_TOGGLE_POWER = "TogglePower"
-
-# Relating to Motorized Shades (MS), Dimmers and Garage Doors
-BOND_DEVICE_ACTION_OPEN = "Open"
-BOND_DEVICE_ACTION_TOGGLE_OPEN = "ToggleOpen"
-BOND_DEVICE_ACTION_CLOSE = "Close"
-BOND_DEVICE_ACTION_HOLD = "Hold"
-BOND_DEVICE_ACTION_PAIR = "Pair"
-BOND_DEVICE_ACTION_PRESET = "Preset"
-
-# Relating to Ceiling Fan (CF)
-BOND_DEVICE_ACTION_SET_SPEED = "SetSpeed"
-BOND_DEVICE_ACTION_INCREASE_SPEED = "IncreaseSpeed"
-BOND_DEVICE_ACTION_DECREASE_SPEED = "DecreaseSpeed"
-
-BOND_DEVICE_ACTION_TURN_LIGHT_ON = "TurnLightOn"
-BOND_DEVICE_ACTION_TURN_LIGHT_OFF = "TurnLightOff"
-BOND_DEVICE_ACTION_TOGGLE_LIGHT = "ToggleLight"
-
-# Relating to Fireplace (FP)
-BOND_DEVICE_ACTION_SET_FLAME = "SetFlame"
-BOND_DEVICE_ACTION_INCREASE_FLAME = "IncreaseFlame"
-BOND_DEVICE_ACTION_DECREASE_FLAME = "DecreaseFlame"
-
+from bond.const import (Actions,DeviceTypes,Directions,Brightness)
```

Comment:
Does this pass the pep? I thought we needed spaces after `,`.

Reply:
Do you run black on this repo?